### ISSUE TITLE
Notifier doesn't show unread emails from multiple accounts

### ISSUE THREAD

**Issue by bermudu** [inbasic/ignotifier#560]
The Notifier used to show all unread messaged for all Gmail accounts. I could've seen all my Gmail account in the left upper corner but now it doesn't. The Notifier shows unread messages only for 1 Gmail account. Any ideas?

**Comment by karama300** [comment]
Same on Firefox 89.0!

**Comment by LoganDark** [comment]
Yeah okay. Fed up with this issue, switching to another extension.

**Comment by LoganDark** [comment]
> Yeah okay. Fed up with this issue, switching to another extension.

"Checker Plus for Gmail" works really well if you don't mind the constant distractions and reminders to follow & donate to the dev.
